public/privacy.html

```
@@ -0,0 +1,163 @@
+<title>プライバシーポリシー | てくしぇあ</title>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)